_content/_layouts/main.njk

```diff
@@ -11,7 +11,7 @@
     </style>
   </head>
 <body>
-{% include "_layouts/nav.njk" %}
+{% include "./nav.njk" %}
```

Comment:
Was easier to just make this a relative include.
Also, I feel like this should be in some `_includes/` folder rather that the `_layouts/` folder, but 🤷‍♂.